src/components/table/TableHeader.tsx

```diff
@@ -1,10 +1,17 @@
-import { Checkbox, TableCell, TableHead, TableRow } from '@mui/material';
+import { Checkbox, TableCell, TableHead, TableRow, Theme } from '@mui/material';
 import React from 'react';
 import { SortOrder } from './sort';
 import { TableColumnType } from './types';
 import { SortableContext } from '@dnd-kit/sortable';
 import TableHeaderItem from './TableHeaderItem';
 import { HeadCell } from '.';
+import { makeStyles } from '@mui/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  headerCell: {
+    borderBottom: `2px solid ${theme.palette.TwClrBrdrSecondary} !important`,
```

Comment:
we may be able to override the `!important` by looking at what class hierarchy is needed to override the style.
eg.
```
headerCell: {
  '& .somereqdclassname': {
     borderBottom: `2px solid ${theme.palette.TwClrBrdrSecondary}`,
  }
}
```